### DISCUSSION TITLE
GitHub Pages is not updating SO FAR (at least 1 day passed and nothing)

### DISCUSSION THREAD

**Discussion by hydroper** [community/community#55161] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I updated my GitHub Pages repository and the changes weren't reflected SO FAR. I actually have two repositories that push changes to GitHub Pages: `hydroper.github.io` and `tictactoe`. Only one branch is being used (`master`).

I'll end up showing the something irrelevant, but here it goes.

Right now I'm unable to see the cv.html file updated:

- [See Source](https://github.com/hydroper/hydroper.github.io/blob/master/cv.html)
- [See current GitHub Pages rendering](https://hydroper.github.io/cv.html)

It's just lacking the last "Tic Tac Toe" card.

## Cache

As of this writting: yesterday I waited around 3 minutes and cleared the cache and nothing yet. Today, after at least 10 hours, I returned there and nothing yet. Again, I cleared the cache and nothing yet.

I think it's an issue with the GitHub action? I'm not very familiar with it yet.

**Comment by hydroper** [comment]
https://webapps.stackexchange.com/q/170557/311020

**Comment by hydroper** [comment]
It's reported that the deploy was successful, but the Artifact Exchange URL says there are errors:

```json
{
  "$id": "1",
  "innerException": null,
  "message": "The user 'System:PublicAccess;aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaaa' is not authorized to access this resource.",
  "typeName": "Microsoft.TeamFoundation.Framework.Server.UnauthorizedRequestException, Microsoft.TeamFoundation.Framework.Server",
  "typeKey": "UnauthorizedRequestException",
  "errorCode": 0,
  "eventId": 3000
}
```

**Comment by hydroper** [comment]
Also got:

> Canceling since a higher priority waiting request for 'pages build and deployment @ master' exists

**Comment by hydroper** [comment]
I got a suggestion from StackExchange to try to push some commit again, so I did that and it finally worked. It's pretty annoying though.

**Comment by GraysonBell** [comment]
The solution that worked for me was go through every component and file and fix every problem and suggestion that showed in the terminal. This updated my deployed pages instantly.
